### ISSUE TITLE
Docker Desktop support

### ISSUE THREAD

**Issue by JustinChristensen** [justincormack/nsenter1#11]
Not to sound ungrateful, because this is really really awesome, and I appreciate your work, but is there something preventing Docker Desktop from shipping with a CLI that handles this?

As far as I can tell there's been a few subpar approaches to getting a TTY with the docker host VM in OSX in the time since Docker Desktop was released, and plenty of open and closed issues in the Docker Desktop for Mac issue tracker. It seems like adding a CLI to do this would be a no brainer (and maybe it is, and you guys have it on your TODO list?).

**Comment by danieldunderfelt** [comment]
I fully agree that there should be a better way to do this with Docker Desktop. I am working with Elasticsearch, and I need to set max_map_count in the vm for the Elastic docker images to work at all. The internet is full of instructions for using `screen`, but it doesn't work with newer Docker desktop versions.

This Docker image is _the only way_ to make it work, but it is not perfect since I can't set persistent sysctl values. Me and my colleagues had to look for a long time to find it. It would be much better if Docker Desktop had a button in the app that just opens a shell where you can do that you need to.

**Comment by justincormack** [comment]
To request a feature, like persistent host sysctl support in Docker Desktop please open an issue on https://github.com/docker/roadmap/issues

**Comment by theodoresi** [comment]
> I fully agree that there should be a better way to do this with Docker Desktop. I am working with Elasticsearch, and I need to set max_map_count in the vm for the Elastic docker images to work at all. The internet is full of instructions for using `screen`, but it doesn't work with newer Docker desktop versions.
> 
> This Docker image is _the only way_ to make it work, but it is not perfect since I can't set persistent sysctl values. Me and my colleagues had to look for a long time to find it. It would be much better if Docker Desktop had a button in the app that just opens a shell where you can do that you need to.

In case someone encounter the same issue, the guidance from [ElasticSearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html#_set_vm_max_map_count_to_at_least_262144) to use docker-machine doesn't work, do this instead
```shell
docker run -it --rm --privileged --pid=host justincormack/nsenter1
sysctl -w vm.max_map_count=262144
```
